ui/lib/core/addon/templates/components/masked-input.hbs

```diff
@@ -1,35 +1,41 @@
-<div class="masked-input {{if shouldObscure "masked"}} {{if displayOnly "display-only"}} {{if allowCopy "allow-copy"}}"
+
+<div class="masked-input {{if displayOnly "display-only"}} {{if allowCopy "allow-copy"}}"
   data-test-masked-input data-test-field>
   {{#if displayOnly}}
-    <pre class="masked-value display-only is-word-break">{{displayValue}}</pre>
-  {{else if maskWhileTyping}}
+    <pre class="masked-value display-only is-word-break {{unless showValue "masked-font"}}">{{unless showValue (truncate value 20) value}}</pre>
```

Comment:
truncate for when looking at very long displayOnly secrets.  E.g. certificates.
![truncate](https://user-images.githubusercontent.com/6618863/115053383-4edb1700-9e9c-11eb-87a9-e8f8d444c4e2.gif)